frameworks/Python/API-Hour/benchmark_config

```diff
@@ -17,11 +17,65 @@
       "language": "Python",
       "orm": "Raw",
       "platform": "API-Hour",
-      "webserver": "None",
+      "webserver": "Gunicorn",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "API-Hour",
-      "notes": "Python 3.4 + AsyncIO"
+      "display_name": "API-Hour+aiohttp.web+pgsql",
+      "notes": "Python 3 + API-Hour + AsyncIO + aiohttp.web + PostgreSQL"
+    },
+    "redis": {
+      "setup_file": "setup",
+      "db_url": "/db_redis",
+      "query_url": "/queries_redis?queries=",
+      "fortune_url": "/fortunes_redis",
+      "update_url": "/updates_redis?queries=",
+      "port": 8008,
+      "approach": "Realistic",
+      "classification": "Micro",
+      "database": "Redis",
+      "framework": "aiohttp.web",
+      "language": "Python",
+      "orm": "Raw",
+      "platform": "API-Hour",
+      "webserver": "Gunicorn",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "API-Hour+aiohttp.web+redis",
+      "notes": "Python 3 + API-Hour + AsyncIO + aiohttp.web + Redis"
+    },
+    "json": {
+      "setup_file": "setup",
+      "json_url": "/json",
+      "port": 8009,
+      "approach": "Stripped",
+      "classification": "Platform",
+      "database": "None",
+      "framework": "AsyncIO",
+      "language": "Python",
+      "orm": "Raw",
+      "platform": "API-Hour",
+      "webserver": "Gunicorn",
+      "os": "Linux",
+      "database_os": "None",
+      "display_name": "API-Hour+AsyncIO",
+      "notes": "Python 3 + API-Hour + AsyncIO"
+    },
+    "plaintext": {
+      "setup_file": "setup",
+      "plaintext_url": "/plaintext",
+      "port": 8011,
+      "approach": "Stripped",
+      "classification": "Platform",
+      "database": "None",
+      "framework": "AsyncIO",
+      "language": "Python",
+      "orm": "Raw",
+      "platform": "API-Hour",
+      "webserver": "Gunicorn",
+      "os": "Linux",
+      "database_os": "None",
+      "display_name": "API-Hour+AsyncIO",
+      "notes": "Python 3 + API-Hour + AsyncIO"
     }
```

Comment:
1. I got plaintext and json running by setting database_os to "Linux" (the error I was getting was `OS or Database OS specified in benchmark_config does not match the current environment. Skipping.`). Maybe we should consider allowing the option "None" to pass through for this sort of purpose, but for now I suggest setting database_os to "Linux."
2. Is there are reason that plaintext and json are completely different tests? I don't really understand why they can't just be combined into one called "stripped," like:

```json
    "stripped": {
      "setup_file": "setup",
      "plaintext_url": "/plaintext",
      "json_url": "/json",
      "port": 8009,
      "approach": "Stripped",
      "classification": "Platform",
      "database": "None",
      "framework": "AsyncIO",
      "language": "Python",
      "orm": "Raw",
      "platform": "API-Hour",
      "webserver": "Gunicorn",
      "os": "Linux",
      "database_os": "Linux",
      "display_name": "API-Hour+AsyncIO",
      "notes": "Python 3 + API-Hour + AsyncIO"
    }
```